electricpy/constants.py

```diff
@@ -12,179 +12,235 @@
+    ]
+)
 # Define Clarke Component Matricies
-Cabc = _np.sqrt(2/3) * _np.array([[ 1, -1/2, -1/2],         # Convert ABC to alpha/beta/gamma
-                                  [ 0, _np.sqrt(3)/2, -_np.sqrt(3)/2],
-                                  [ 1/_np.sqrt(2), 1/_np.sqrt(2), 1/_np.sqrt(2)]])
-Cxyz = _np.array([[ 2/_np.sqrt(6), 0, 1/_np.sqrt(3)],       # Convert alpha/beta/gamma to ABC
-                  [ -1/_np.sqrt(6), 1/_np.sqrt(2), 1/_np.sqrt(3)],
-                  [ -1/_np.sqrt(6), -1/_np.sqrt(2), 1/_np.sqrt(3)]])
+Cabc = _np.sqrt(2 / 3) * _np.array(
+    [
+        [1, -1 / 2, -1 / 2],  # Convert ABC to alpha/beta/gamma
+        [0, _np.sqrt(3) / 2, -_np.sqrt(3) / 2],
+        [1 / _np.sqrt(2), 1 / _np.sqrt(2), 1 / _np.sqrt(2)],
+    ]
+)
+Cxyz = _np.array(
+    [
+        [2 / _np.sqrt(6), 0, 1 / _np.sqrt(3)],  # Convert alpha/beta/gamma to ABC
+        [-1 / _np.sqrt(6), 1 / _np.sqrt(2), 1 / _np.sqrt(3)],
+        [-1 / _np.sqrt(6), -1 / _np.sqrt(2), 1 / _np.sqrt(3)],
+    ]
+)
 # Define Park Components Matricies
-_rad = lambda th: _np.radians( th )
-Pdq0_im = lambda th: _np.sqrt(2/3)*_np.array([[ _np.cos(_rad(th)), _np.cos(_rad(th)-2*pi/3), _np.cos(_rad(th)+2*pi/3)],
-                                           [-_np.sin(_rad(th)),-_np.sin(_rad(th)-2*pi/3),-_np.sin(_rad(th)+2*pi/3)],
-                                           [ _np.sqrt(2)/2,     _np.sqrt(2)/2,            _np.sqrt(2)/2]])
-Pabc_im = lambda th: _np.sqrt(2/3)*_np.array([[ _np.cos(_rad(th)),      -_np.sin(_rad(th)),        _np.sqrt(2)/2],
-                                           [_np.cos(_rad(th)-2*pi/3),-_np.sin(_rad(th)-2*pi/3), _np.sqrt(2)/2],
-                                           [_np.cos(_rad(th)+2*pi/3),-_np.sin(_rad(th)+2*pi/3), _np.sqrt(2)/2]])
-Pdq0 = 2/3 * _np.array([[0,-_np.sqrt(3/2),_np.sqrt(3/2)],
-                        [1,-1/2,-1/2],
-                        [1/2, 1/2, 1/2]])
-Pqd0 = 2/3 * _np.array([[1,-1/2,-1/2],
-                        [0,-_np.sqrt(3/2),_np.sqrt(3/2)],
-                        [1/2, 1/2, 1/2]])
-                 
+_rad = lambda th: _np.radians(th)
+Pdq0_im = lambda th: _np.sqrt(2 / 3) * _np.array(
+    [
+        [
+            _np.cos(_rad(th)),
+            _np.cos(_rad(th) - 2 * pi / 3),
+            _np.cos(_rad(th) + 2 * pi / 3),
+        ],
+        [
+            -_np.sin(_rad(th)),
+            -_np.sin(_rad(th) - 2 * pi / 3),
+            -_np.sin(_rad(th) + 2 * pi / 3),
+        ],
+        [_np.sqrt(2) / 2, _np.sqrt(2) / 2, _np.sqrt(2) / 2],
+    ]
+)
+Pabc_im = lambda th: _np.sqrt(2 / 3) * _np.array(
+    [
+        [_np.cos(_rad(th)), -_np.sin(_rad(th)), _np.sqrt(2) / 2],
+        [
+            _np.cos(_rad(th) - 2 * pi / 3),
+            -_np.sin(_rad(th) - 2 * pi / 3),
+            _np.sqrt(2) / 2,
+        ],
+        [
+            _np.cos(_rad(th) + 2 * pi / 3),
+            -_np.sin(_rad(th) + 2 * pi / 3),
+            _np.sqrt(2) / 2,
+        ],
+    ]
+)
+Pdq0 = (
+    2
+    / 3
```

Comment:
- [ ] there must be another way to clean this up and make it more visually cohesive so it'll be easier to maintain.